CatAnalyzer/plugins/TtbarSingleLeptonAnalyzer.cc

```diff
@@ -221,8 +244,6 @@ void TtbarSingleLeptonAnalyzer::analyze(const edm::Event& iEvent, const edm::Eve
   b_Jet_py = new std::vector<float>;
   b_Jet_pz = new std::vector<float>;
   b_Jet_E  = new std::vector<float>;
-
-  b_Jet_LooseID = new std::vector<bool>;
   
   b_Jet_partonFlavour = new std::vector<int>;
   b_Jet_hadronFlavour = new std::vector<int>;
```

Comment:
Do the memory allocation in the constructor (and delete in the destructor).
call b_Jet_py->clear() here.